signs-work-order-map/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["next/core-web-vitals", "next/typescript", "prettier"],
```

Comment:
when i run `npm run lint` i get an error about a missing prettier config 💥

Reply:
I added the eslint plugins like vz. now im getting an error message when i run `npm run lint`
`ESLint configuration in ../package.json » eslint-config-react-app is invalid:`, I'll see where the mismatch is.